docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,234 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: HTTP client and server
+--->
+
+# Semantic Conventions for HTTP client and server metrics emitted by .NET
+
+**Status**: [Stable][DocumentStatus]
+
+This article defines semantic conventions for HTTP metrics emitted by .NET components and runtime.
+
+<!-- toc -->
+
+- [HTTP client](#http-client)
+  * [Metric: `http.client.request.duration`](#metric-httpclientrequestduration)
+  * [Metric: `http.client.open_connections`](#metric-httpclientopen_connections)
+  * [Metric: `http.client.connection.duration`](#metric-httpclientconnectionduration)
+  * [Metric: `http.client.request.time_in_queue`](#metric-httpclientrequesttime_in_queue)
+  * [Metric: `http.client.active_requests`](#metric-httpclientactive_requests)
+- [HTTP server](#http-server)
+  * [Metric: `http.server.request.duration`](#metric-httpserverrequestduration)
+  * [Metric: `http.server.active_requests`](#metric-httpserveractive_requests)
+
+<!-- tocstop -->
+
+## HTTP client
+
+All Http client metrics are reported by the `System.Net.Http` meter.
+
+### Metric: `http.client.request.duration`
+
+Client request duration measures the time it takes to receive response headers and doesn't include time to read the response body.
+
+This metric follows the common [http.client.request.duration](../http/http-metrics.md#metric-httpclientrequestduration) definition.
+
+Notes:
+
+- Meter name is `System.Net.Http`
+- Metric added in .NET 8.0
+- When the `error.type` attribute is reported, it contains one of [HTTP Request errors](https://learn.microsoft.com/dotnet/api/system.net.http.httprequesterror) in snake_case, a full exception type, or a string representation of received status code.
+- `network.protocol.name` is not reported and should always be assumed to match `http`.
+- `server.port` is not reported when it matches a default one for provided scheme (`443` for `https` and `80` for `http`)
+- `url.scheme` is always reported.
```

Comment:
The last two were one of the latest changes in HTTP semconv and now .NET is not fully aligned - created https://github.com/dotnet/runtime/issues/94829 to track it